README.md

```diff
@@ -95,8 +102,25 @@ jobs:
         with:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
 
-      - name: blue green deploy step
-        uses: azure/spring-cloud-git-action@v1
+      - name: deploy to production step with soruce code
+        uses: azure/spring-cloud-deploy@v1
+        with:
+          azure-subscription: ${{ env.AZURE_SUBSCRIPTION }}
+          action: Deploy
+          service-name: <service instance name>
+          app-name: <app name>
+          use-staging-deployment: false
+          deployment-name: default
+          package: ${{ env.ASC_PACKAGE_PATH }}
+```
+
+#### Blue-green
+
+The following examples deploys to a pre-existing staging deployment. This deployment will not receive production traffic until it is set as a production deployment. You can set use-staging-deployment true to find the staging deployment automatically or just allocate specific deployment-name. We will only focus on the spring-cloud-deploy action and leave out the preparatory jobs in the rest of the article.
```

Comment:
examples -> example? or deploys -> deploy.

Reply:
to an existing staging deployment.

Reply:
I think this can be removed. `We will only focus on the spring-cloud-deploy action and leave out the preparatory jobs in the rest of the article.`

Reply:
We will leave out some jobs later because of duplicate content. Need we point out that or is there any better notification?